src/tests/ftest/erasurecode/restart.py

```diff
@@ -61,95 +87,97 @@ def execution(self, agg_check=None):
-                    "transfer_size": block_transfer_sizes[1],
-                    "block_size": block_transfer_sizes[0]
-                }
+        ior_kwargs = {
+            "test": self,
+            "manager": job_manager,
+            "log": None,
+            "hosts": self.hostlist_clients,
+            "path": self.workdir,
+            "slots": None,
+            "group": self.server_group,
+            "pool": self.pool,
+            "container": None,
+            "processes": processes,
+            "ppn": ppn,
+            "intercept": None,
+            "plugin_path": None,
+            "dfuse": None,
+            "display_space": True,
+            "fail_on_warning": False,
+            "namespace": "/run/ior/*",
+            "ior_params": {
+                "dfs_oclass": None,
+                "dfs_dir_oclass": None,
+                "flags": ior_w_flags,
+                "transfer_size": transfer_size,
+                "block_size": block_size
             }
-            try:
-                run_ior(**ior_kwargs)
-            except CommandFailure as error:
-                self.fail("#IOR write failed, {}".format(error))
+        }
+        self.run_ior_containers(containers, ior_kwargs)
 
         # 5.
         self.log_step("Check for free space after IOR write all EC object data to container")
         free_space_after_ior = self.pool.get_total_free_space(refresh=True)
         space_used_by_ior = initial_free_space - free_space_after_ior
-        self.log.info("=After 1st IOR write, pool free space: %s", free_space_after_ior)
-        self.log.info("=Space used by 1st IOR: %s", space_used_by_ior)
+        expected_free_aggr = int(free_space_after_ior - space_used_by_ior * aggr_threshold)
+        self.log.info("=After 1st IOR write, pool free space: %s", f"{free_space_after_ior:,}")
+        self.log.info("=Space used by 1st IOR: %s", f"{space_used_by_ior:,}")
         self.assertLess(free_space_after_ior, initial_free_space,
                         "Pool free space has not been reduced by IOR writes")
 
         # 6.
         self.log_step("Run IOR write again with the same data to each container")
-        for container in containers:
-            ior_kwargs["container"] = container
-            try:
-                run_ior(**ior_kwargs)
-            except CommandFailure as error:
-                self.fail("#IOR write failed, {}".format(error))
+        self.run_ior_containers(containers, ior_kwargs)
         free_space_after_second_ior = self.pool.get_total_free_space(refresh=True)
         space_used_by_2nd_ior = free_space_after_ior - free_space_after_second_ior
         self.log.info("=After 2nd IOR write with same data, pool free space: %s",
-                      free_space_after_second_ior)
-        self.log.info("=Space used by 1st IOR: %s", space_used_by_ior)
-        self.log.info("=Space used by 2nd IOR: %s", space_used_by_2nd_ior)
+                      f"{free_space_after_second_ior:,}")
```

Comment:
What does the `:,` syntax do?

Reply:
The syntax is to display a format with "," of pool free space, to make it more readable for a large integer.